src/rust/src/backend/kdf.rs

```diff
@@ -517,6 +540,265 @@ impl Argon2id {
     }
 }
 
+#[pyo3::pyclass(module = "cryptography.hazmat.primitives.kdf.argon2")]
+struct Argon2d {
+    #[allow(unused)]
+    base: BaseArgon2,
```

Comment:
Let's just name this `_base` and then we don't need the annotation (same for all the other structures variants)